test/features/capture_errors.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# Even though the early deploy feature isnt a direct requirement to the error capture,
+# it adds more points where exception might be raised. So it makes sense to leave it
+# enabled
+Syskit.conf.early_deploy = true
+Syskit.conf.capture_errors_during_network_resolution = true
```

Comment:
Decouple the two flags, and run 4 test suites (plain core, with early deploy, with capture errors and with both)